spot-client/src/spot-tv/ui/views/spot-view.js

```diff
@@ -4,9 +4,6 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
 
-
-import { JoinInfo } from './../components';
```

Comment:
Is this component used elsewhere? If not, please kill it.

Reply:
It's still used on the [/tv page](https://github.com/jitsi/jitsi-meet-spot/blob/master/spot-client/src/spot-tv/ui/views/home.js#L124)